refinery/data_set_manager/tests.py

```diff
@@ -1502,6 +1507,23 @@ def test_get_relative_file_store_item_url(self):
         self.assertEqual(relative_url, self.file_node.get_file_store_item(
         ).get_datafile_url())
 
+    def test_get_analysis(self):
+        make_analyses_with_single_dataset(1, self.user)
+        self.analysis = Analysis.objects.all()[0]
```

Comment:
inside a test, I'm not sure there's a good reason to use fields instead of plain variables...